server/seeders/20190309124955-users.js

```diff
@@ -1,40 +1,97 @@
-'use strict';
+"use strict";
 
 const faker = require("faker");
 
 module.exports = {
-    up: (queryInterface, Sequelize) => {
-        const newData = [];
+  up: (queryInterface, Sequelize) => {
+    const newData = [];
 
-        for (let i = 0; i < 200; i++) {
-            const seedData = {
-                username: faker.name.findName(),
-                password: faker.name.findName(),
-                createdAt: new Date(),
-                updatedAt: new Date()
-            };
-            newData.push(seedData);
-        }
+    for (let i = 0; i < 200; i++) {
+      const seedData = {
+        username: faker.internet.userName(),
+        password: faker.internet.password(),
+        email: faker.internet.email(),
+        createdAt: new Date(),
+        updatedAt: new Date()
+      };
+      newData.push(seedData);
+    }
 
-        const accounts = [
-            {id: 1000001, username: "docent", password: "docent", createdAt: new Date(), updatedAt: new Date()},
-            {id: 1000002, username: "onderwijsprogrammacoordinator", password: "onderwijsprogrammacoordinator", createdAt: new Date(), updatedAt: new Date()},
-            {id: 1000003, username: "projectcoordinator", password: "projectcoordinator", createdAt: new Date(), updatedAt: new Date()},
-            {id: 1000004, username: "modulecoordinator", password: "modulecoordinator", createdAt: new Date(), updatedAt: new Date()},
-            {id: 1000005, username: "facilitator", password: "facilitator", createdAt: new Date(), updatedAt: new Date()},
-            {id: 1000006, username: "admin", password: "admin", createdAt: new Date(), updatedAt: new Date()},
-            {id: 1000007, username: "roosteraar", password: "roosteraar", createdAt: new Date(), updatedAt: new Date()},
-            {id: 1000008, username: "coordinator", password: "coordinator", createdAt: new Date(), updatedAt: new Date()}
-        ];
```

Comment:
ik had per rij gedaan voor readability en compactheid want nu hebben we ipv 8, 60 rijen maar is een kwestie van voorkeur

Reply:
Weer een format on save dingetje.